server.js

```diff
@@ -7,9 +7,30 @@ const getFiles = async ()=>{
     return [indexHtml,indexJs]
 }
 
-
 http.createServer(async ({ url }, res) => {
     const [indexHtml, indexJs] = await getFiles()
+
+    if (url.includes('value')) {
+        const inputValue = url.match(/=(\w+(?:%20\w+)*)/g)
+        if (inputValue === null) {
+            return 
```

Comment:
Because of this line, request never ends.

Reply:
Removed :)